Doesn't build against Kea 1.6

Good Day,

Installed Kea from the package repo at cloudsmith.

Attempted to compile this hook and got "kea-msg-compiler: command not found", seems this is no longer included in 1.6 to get it I had to download the 1.5 tarball and compile to get the command (after trying the 1.6 tarball and compiling that).

Attempting to compile against the 1.6 headers yields the following:

> airlink@dhcp001:~/kea-hook-runscript$ make
/home/airlink/kea-1.5.0/src/lib/log/compiler/kea-msg-compiler -d src/ src/messages.mes
touch s-messages
g++ -MMD -MP -c -I /usr/include/kea -fPIC -Wno-deprecated -std=c++11 -o src/messages.o src/messages.cc
g++ -MMD -MP -c -I /usr/include/kea -fPIC -Wno-deprecated -std=c++11 -o src/logger.o src/logger.cc
g++ -MMD -MP -c -I /usr/include/kea -fPIC -Wno-deprecated -std=c++11 -o src/load.o src/load.cc
g++ -MMD -MP -c -I /usr/include/kea -fPIC -Wno-deprecated -std=c++11 -o src/runscript.o src/runscript.cc
g++ -MMD -MP -c -I /usr/include/kea -fPIC -Wno-deprecated -std=c++11 -o src/callouts.o src/callouts.cc
src/callouts.cc: In function ‘void extract_lease4(std::vector<std::__cxx11::basic_string<char> >&, isc::dhcp::Lease4Ptr)’:
src/callouts.cc:158:69: error: ‘struct isc::dhcp::Lease4’ has no member named ‘t1_’
     env.push_back("KEA_LEASE4_RENEW_TIMER=" + std::to_string(lease->t1_));
                                                                     ^~~
src/callouts.cc:159:70: error: ‘struct isc::dhcp::Lease4’ has no member named ‘t2_’
     env.push_back("KEA_LEASE4_REBIND_TIMER=" + std::to_string(lease->t2_));
                                                                      ^~~
src/callouts.cc: In function ‘void extract_lease6(std::vector<std::__cxx11::basic_string<char> >&, isc::dhcp::Lease6Ptr)’:
src/callouts.cc:178:69: error: ‘struct isc::dhcp::Lease6’ has no member named ‘t1_’
     env.push_back("KEA_LEASE6_RENEW_TIMER=" + std::to_string(lease->t1_));
                                                                     ^~~
src/callouts.cc:179:70: error: ‘struct isc::dhcp::Lease6’ has no member named ‘t2_’
     env.push_back("KEA_LEASE6_REBIND_TIMER=" + std::to_string(lease->t2_));
                                                                      ^~~
Makefile:15: recipe for target 'src/callouts.o' failed
make: *** [src/callouts.o] Error 1

Compiling against the 1.5 code works and generates the kea-hook-runscript.so file but it doesn't load into the 1.6 server. Error is as follows:

> DHCP4_CONFIG_LOAD_FAIL configuration error using file: /etc/kea/kea-dhcp4.conf, reason: hooks libraries failed to validate

--- a/src/callouts.cc
+++ b/src/callouts.cc
@@ -155,8 +155,6 @@ void extract_lease4(std::vector<std::string>& env, const Lease4Ptr lease)
     env.push_back("KEA_LEASE4_HWADDR=" + lease->hwaddr_->toText(false));
     env.push_back("KEA_LEASE4_HOSTNAME=" + lease->hostname_);
     env.push_back("KEA_LEASE4_CLIENT_LAST_TRANSMISSION=" + std::to_string(lease->cltt_));
-    env.push_back("KEA_LEASE4_RENEW_TIMER=" + std::to_string(lease->t1_));
-    env.push_back("KEA_LEASE4_REBIND_TIMER=" + std::to_string(lease->t2_));
     env.push_back("KEA_LEASE4_VALID_LIFETIME=" + std::to_string(lease->valid_lft_));
     env.push_back("KEA_LEASE4_DEBUG=" + lease->toText());
 }
@@ -175,8 +173,6 @@ void extract_lease6(std::vector<std::string>& env, const Lease6Ptr lease)
     env.push_back("KEA_LEASE6_HOSTNAME=" + lease->hostname_);
     env.push_back("KEA_LEASE6_CLIENT_DUID=" + lease->duid_->toText());
     env.push_back("KEA_LEASE6_CLIENT_LAST_TRANSMISSION=" + std::to_string(lease->cltt_));
-    env.push_back("KEA_LEASE6_RENEW_TIMER=" + std::to_string(lease->t1_));
-    env.push_back("KEA_LEASE6_REBIND_TIMER=" + std::to_string(lease->t2_));
     env.push_back("KEA_LEASE6_VALID_LIFETIME=" + std::to_string(lease->valid_lft_));
     env.push_back("KEA_LEASE6_PREFERRED_LIFETIME=" + std::to_string(lease->preferred_lft_));
     env.push_back("KEA_LEASE6_IAID=" + std::to_string(lease->iaid_));

